modules/administration-guide/partials/proc_setting-up-the-github-oauth-app.adoc

```diff
@@ -0,0 +1,52 @@
+:_content-type: PROCEDURE
+:description: Setting up the GitHub OAuth App
+:keywords: github, github-oauth-app
+:navtitle: Setting up the GitHub OAuth App
+// :page-aliases:
+
+[id="setting-up-the-github-oauth-app_{context}"]
+= Setting up the GitHub OAuth App
+
+Set up a GitHub OAuth App using OAuth 2.0.
+
+.Prerequisites
+
+* You are logged in to GitHub.
+* link:https://www.gnu.org/software/coreutils/base64[`base64`] is installed in the operating system you are using.
+
+.Procedure
+
+. Go to link:https://github.com/settings/applications/new[].
+//Long version: Click your GitHub avatar menu:Settings[Developer settings > OAuth Apps > Register a new application]. max-cx
+
+. Enter *{prod-short}* as the *Application name*.
+
+. Enter `pass:c,a,q[{prod-url}]/` as the *Homepage URL*.
+
+. Enter `pass:c,a,q[{prod-url}]/api/oauth/callback` as the *Authorization callback URL*.
+
+. Click *Register application*.
+
+. Click *Generate new client secret*.
+
+. Copy the *GitHub OAuth Client ID* and encode it to Base64:
```

Comment:
Consider merging all remaining step into one single step:

* Convert *GitHub OAuth Client ID* and *GitHub OAuth Client Secret* values to the Base64 format for later use in {prod-short}.

Reply:
Thanks for the suggestion.
I've merged two the two steps, yet differently:

![image](https://user-images.githubusercontent.com/78564304/163004925-617e8d0c-836e-4f51-8ca6-f944b03a5b24.png)